rules/windows/process_creation/proc_creation_win_wmic_system_info_discovery.yml

```diff
@@ -0,0 +1,40 @@
+title: System Information Discovery with WMIC
+id: 9d5a1274-922a-49d0-87f3-8c653483b909
+status: experimental
+description: Detect use of the WMI command-line (WMIC) utility to identify and display various system information, including OS, CPU, GPU, and disk drive names; memory capacity; display resolution; and baseboard, BIOS, and GPU driver products/versions. Some of these commands were used by Aurora Stealer in late 2022/early 2023.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/a2ccd19c37d0278b4ffa8583add3cf52060a5418/atomics/T1082/T1082.md#atomic-test-25---system-information-discovery-with-wmic
+    - https://nwgat.ninja/getting-system-information-with-wmic-on-windows/
+    - https://blog.sekoia.io/aurora-a-rising-stealer-flying-under-the-radar
+    - https://blog.cyble.com/2023/01/18/aurora-a-stealer-using-shapeshifting-tactics/
+    - https://app.any.run/tasks/a6aa0057-82ec-451f-8f99-55650ca537da/
+author: TropChaud
+date: 2023/01/26
+tags:
+    - attack.discovery
+    - attack.t1082
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_wmic:
+        - Description: 'WMI Commandline Utility'
+        - OriginalFileName: 'wmic.exe'
+        - Image|endswith: '\WMIC.exe'
+        - CommandLine|contains: 'wmic'
+    selection_commands:
+        CommandLine|contains:
+            - 'cpu get name'
+            - 'MEMPHYSICAL get MaxCapacity'
+            - 'baseboard get product'
+            - 'baseboard get version'
+            - 'bios get SMBIOSBIOSVersion'
+            - 'path win32_VideoController get name'
+            - 'path win32_VideoController get DriverVersion'
+            - 'path win32_VideoController get VideoModeDescription'
+            - 'OS get Caption,OSArchitecture,Version'
+            - 'DISKDRIVE get Caption'
```

Comment:
I'm not too fond of the rule for a couple of reasons.

- It's too specific from a command line order. Even though it's been used by malware as the references shows
- The commands are unfortunately not "suspicious" by nature and non-malware related. In fact, they are often used by software such as video games to get information about video cards or get the max memory. Other software getting CPU...etc.
- It different commands targeting different things in one rule.

This rule would work in my opinion as a correlation rule where the execution of such commands happened in succession and with a certain execution rate.

You could write it as a SIGMA correlation but unfortunately, the old notation is EOL and the new one from PySIGMA isn't yet in "prod".

Other opinions from @frack113 @Neo23x0 @phantinuss  could help.

Reply:
The correlation idea makes sense - I originally considered that then decided to change simply to align with the Atomic test. But your comments make complete sense.

Reporting and samples indicate the stealer only uses three of these commands (os get Caption, path win32_VideoController get name, cpu get name), and in the samples I've reviewed, execution of those always occurred rapidly (within ~1 second).